### ISSUE TITLE
Add `--init` to automatically bootstrap configuration or add local config generation to `init`

### ISSUE THREAD

**Issue by jsoverson** [apexlang/cli#5]
Many projects that require configuration have an `--init` switch or `init` subcommand that – by default – generates a boilerplate configuration file and/or support files.

e.g. a command like this...

```shell-session
$ apex --init
Initializing new apex.yaml and schema.apexlang...
```

...would generate an `apex.yaml` (if it doesn't exist)...

```yaml
spec: schema.apexlang
generates:
  default.txt
    module: "@apexlang/codegen/html"
```

...and a `schema.apexlang` like this (if it doesn't exist).

```apexlang
namespace "project.v1"

"NewService interface for querying data"
interface NewService {
  query(id: u32) [Order]
}

"Type for Order records"
type Order {
  id: u32
  data: value
}
```

**Comment by pkedy** [comment]
There already is `apex init <template>`.

```
➜  tinygo-test2 apex --help

...

  init <template> [<variables> ...]
    Initializes an existing project directory from a template.
```

**Comment by jsoverson** [comment]
Yes, but it requires a template. I mean for the `apex init` (or `apex --init`) to initialize apex itself (+ a schema) so arbitrary projects can start rapidly without an 'official' template

**Comment by pkedy** [comment]
Perhaps the template is made to have a default value that points to a very simple template

**Comment by pkedy** [comment]
This was fixed as part of #8
